invenio_rdm_records/contrib/imprint/custom_fields.py

```diff
@@ -10,7 +10,6 @@
 - imprint.isbn
 - imprint.pages
 - imprint.place
-- imprint.publisher
```

Comment:
Publisher was redundant, the record already has a publisher. Therefore, it was removed.